src/StockportWebapp/Views/stockroom/Events/Index.cshtml

```diff
@@ -11,8 +11,24 @@
     ViewData["Title"] = ViewData["og:title"];
     ViewData["Description"] = Model.Homepage.MetaDescription;
     Layout = "../Shared/_Layout.cshtml";
+
+    string eventsTitle = "";
```

Comment:
string.Empty, also potentially a double ternary but that might make it less readable